packages/react-server/src/ReactFizzServer.js

```diff
@@ -4636,7 +4629,6 @@ function abortTask(task: Task, request: Request, error: mixed): void {
       }
     }
   } else {
-    boundary.pendingTasks--;
```

Comment:
Since some of the paths below here are going to take the `finishedTask` path and that does all the decrementing we need, we don't need to release anything above it. This is causing an extra task to be released (ending up in negative probably).